### PR TITLE
equalTo + notEqualTo: Add possibility to pass objects

### PR DESCRIPTION
**Pull request by paulovitin**
- Add possibility to pass objects to equalTo and notEqualTo

_@dplewis Please add hacktoberfest label_

### CODE REVIEW

File: src/__tests__/ParseQuery-test.js [hunk 1]
```diff
@@ -130,6 +140,20 @@ describe('ParseQuery', () => {
         }
       }
     });
+
+    const size = 'medium';
+    const stock = true;
+    q.notEqualTo({ size, stock });
+    expect(q.toJSON()).toEqual({
```
Comment (dplewis) [inline src/__tests__/ParseQuery-test.js:147]: Can you add an integration test?

**Review by dplewis — CHANGES_REQUESTED** [review]
Parse Community is already apart of HacktoberFest

File: src/__tests__/ParseQuery-test.js (hunk 1 above)
Comment (paulovitin) [reply src/__tests__/ParseQuery-test.js:147]: Where do I find an integrative test in this file?

Comment (dplewis) [reply src/__tests__/ParseQuery-test.js:147]: `/integration/test/ParseQueryTest.js`

Comment (paulovitin) [reply src/__tests__/ParseQuery-test.js:147]: Done.. ;)

**Review by dplewis — APPROVED** [review]
LGTM!

**Pull request merged**